PromoteLiteMemberTest.promotion_shouldFail_whenMasterIsSuspected_duringPromotion

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-freestyle/369/testReport/junit/com.hazelcast.internal.cluster.impl/PromoteLiteMemberTest/promotion_shouldFail_whenMasterIsSuspected_duringPromotion/

```
java.lang.AssertionError: expected:<[127.0.0.1]:7056> but was:<[127.0.0.1]:7036>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.hazelcast.internal.cluster.impl.MembershipFailureTest.assertMaster(MembershipFailureTest.java:832)
	at com.hazelcast.internal.cluster.impl.PromoteLiteMemberTest$3.run(PromoteLiteMemberTest.java:312)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:992)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1009)
	at com.hazelcast.internal.cluster.impl.PromoteLiteMemberTest.promotion_shouldFail_whenMasterIsSuspected_duringPromotion(PromoteLiteMemberTest.java:309)
```

https://hazelcast-l337.ci.cloudbees.com/job/new-lab-fast-pr/8655/testReport/junit/com.hazelcast.internal.cluster.impl/PromoteLiteMemberTest/promotion_shouldFail_whenMasterIsSuspected_duringPromotion/

```
java.lang.AssertionError: com.hazelcast.core.OperationTimeoutException: PromoteLiteMemberOp invocation failed to complete due to operation-heartbeat-timeout. Current time: 2017-05-04 08:36:52.250. Start time: 2017-05-04 08:34:51.557. Total elapsed time: 120693 ms. Last operation heartbeat: never. Last operation heartbeat from member: never. Invocation{op=com.hazelcast.internal.cluster.impl.operations.PromoteLiteMemberOp{serviceName='hz:core:clusterService', identityHash=2105013238, partitionId=-1, replicaIndex=0, callId=-8, invocationTime=1493886892051 (2017-05-04 08:34:52.051), waitTimeout=-1, callTimeout=60000}, tryCount=250, tryPauseMillis=500, invokeCount=6, callTimeoutMillis=60000, firstInvocationTimeMs=1493886891557, firstInvocationTime='2017-05-04 08:34:51.557', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5021, pendingResponse={VOID}, backupsAcksExpected=0, backupsAcksReceived=0, connection=MockConnection{localEndpoint=[127.0.0.1]:5031, remoteEndpoint=[127.0.0.1]:5021}} is not an instanceof java.lang.IllegalStateException
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertInstanceOf(HazelcastTestSupport.java:770)
	at com.hazelcast.internal.cluster.impl.PromoteLiteMemberTest.promotion_shouldFail_whenMasterIsSuspected_duringPromotion(PromoteLiteMemberTest.java:327)
```

--- a/hazelcast/src/test/java/com/hazelcast/internal/cluster/impl/PromoteLiteMemberTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/internal/cluster/impl/PromoteLiteMemberTest.java
@@ -55,7 +55,6 @@ import static com.hazelcast.internal.cluster.impl.ClusterDataSerializerHook.PROM
 import static com.hazelcast.internal.cluster.impl.MembershipFailureTest.assertMaster;
 import static com.hazelcast.internal.cluster.impl.PacketFiltersUtil.dropOperationsBetween;
 import static com.hazelcast.internal.cluster.impl.PacketFiltersUtil.dropOperationsFrom;
-import static com.hazelcast.internal.cluster.impl.PacketFiltersUtil.resetPacketFiltersFrom;
 import static org.hamcrest.Matchers.greaterThan;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotEquals;
@@ -319,7 +318,7 @@ public class PromoteLiteMemberTest extends HazelcastTestSupport {
             }
         });
 
-        resetPacketFiltersFrom(hz3);
+        dropOperationsBetween(hz3, hz1, EXPLICIT_SUSPICION);
         try {
             future.get();
             fail("Promotion should fail!");

